x/metrix/keeper/keeper.go

```diff
@@ -186,6 +194,11 @@ func (k Keeper) OnSnapshotBuilt(ctx sdk.Context, snapshot *valsettypes.Snapshot)
 			logger.Info("Skip updating metrics, no chains found.")
 			return
 		}
+		scoreMax = int64(len(v.ExternalChainInfos))
+		if scoreMax < 1 {
+			logger.Info("Skip updating metrics, no chains found.")
+			return
+		}
```

Comment:
This should already be on the current master, looks like you're not fully rebased?